custom_components/unifi_insights/__init__.py

```
@@ -27,10 +27,12 @@
     CONF_CONNECTION_TYPE,
     CONF_CONSOLE_ID,
     CONNECTION_TYPE_LOCAL,
-    CONNECTION_TYPE_REMOTE as CONNECTION_TYPE_REMOTE,
     DEFAULT_API_HOST,
     DOMAIN,
 )
+from .const import (
+    CONNECTION_TYPE_REMOTE as CONNECTION_TYPE_REMOTE,
+)
```

Pylint (4.0.7)

[convention] Import alias does not rename original package

(C0414)